PlatformIO/src/devices/Inmp441Max98357a.hpp

```diff
@@ -17,17 +17,28 @@
 // GPIO12 <--> DIN
 #define I2S_LRC 16
 #define I2S_BCLK 13
+
+#if ESP_TYPE == ESP32_POE_ISO
+#define I2S_DIN 3
+#else
 #define I2S_DIN 12
+#endif
+
 #define I2S_PORT_TX I2S_NUM_1
 
 #define I2S_SAMPLE_RATE   (16000)
 #define I2S_SAMPLE_BITS   (16)
 #define I2S_READ_LEN     512
 
+#if ESP_TYPE != ESP32_POE_ISO
```

Comment:
Does not have to be == instead of != ?
It seems to me to master branch has LED 33 and with this if statement LED 5 will be active if ESP_TYPE is not ESP32_POE_ISO

Reply:
yes, you're right. The ESP32-POE-ISO doesn't have LEDs, so I just used dummy pins. That's why I didn't see the error.